### ISSUE TITLE
Apostroph character escaping issue in the profile panel

### ISSUE THREAD

**Issue by damienalexandre** [umputun/remark42#1293]
Using `umputun/remark42:master`, I get a escaping/encoding issue with the `'` character in the profile panel.

```html
<script type="[text/javascript]()">
    var remark_config = {
        host: '*******',
        site_id: '*****',
        components: ['embed'],
        url: 'https://*****/blog/lavantage-de-rouler-avant-tout',
        max_shown_comments: 20,
        theme: 'light',
        page_title: "L'avantage de rouler avant-tout",
        locale: 'fr',
        show_email_subscription: false
    };
</script>
```

The important line here is `page_title: "L'avantage de rouler avant-tout"`, the `'` is displayed like this in the user profile:

![image](https://user-images.githubusercontent.com/225704/157922964-9c429b4f-f8e6-45ab-8622-ac20357f9ded.png)

**Comment by akellbl4** [comment]
@paskal we use `orig` field from a comment. It comes with `&#39;` in it. We do not render text html due to security reason. Could address this change to backend and do not encode this char like so?

**Comment by paskal** [comment]
Thanks for the report! It was fixed in #1341, I've verified it:
<img width="433" alt="image" src="https://user-images.githubusercontent.com/712534/165991255-fa1b799c-4198-4e8e-a97b-41e01a3b9a4b.png">
Resolving.
